setup-gcloud/README.md

```diff
@@ -25,74 +25,3 @@ steps:
 +  uses: google-github-actions/setup-gcloud@master
```

Comment:
I think we can go ahead and delete this file. @bharathkkb do you think we should add a note in the readme about the deprecated actions?

Reply:
Yeah I think we should add a readme note stating these are not deprecated, but rather live elsewhere.

Reply:
@tequilarista I am thinking we can move the deprecation notice to main root readme for increased visibility and get rid of `setup-gcloud` dir completely?

Reply:
makes sense, will update